### DISCUSSION TITLE
Unable to see the dashboard but able to see the charts of the dashboard in apache superset gamma user

### DISCUSSION THREAD

**Discussion by srishtiagrawall** [community/community#138893] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have provided these following permissions to my gamma users:

[can read on Chart, can read on Dashboard, can get on OpenApi, can read on DashboardFilterStateRestApi, can read on Explore, can read on ExploreFormDataRestApi, can read on ExplorePermalinkRestApi, can estimate query cost on SQLLab, datasource access on [examples].[wb_health_population](id:1), can dashboard on Superset, can read on AdvancedDataType, can this form get on ResetMyPasswordView, can userinfo on UserDBModelView, resetmypassword on UserDBModelView, can show on SwaggerView, can list on AsyncEventsRestApi, can invalidate on CacheRestApi, can export on Dashboard, can get embedded on Dashboard, can write on ExploreFormDataRestApi, can write on ExplorePermalinkRestApi, can format sql on SQLLab, can list on DynamicPlugin, can show on DynamicPlugin, can external metadata by name on Datasource, can get value on KV, can log on Superset, can fetch datasource metadata on Superset, can explore json on Superset, can dashboard permalink on Superset, can slice on Superset, can recent activity on Log, can read on SecurityRestApi, can read on RowLevelSecurity, can read on EmbeddedDashboard, schema access on [MySQL].[eeyedata]]

I don't want to share any data with them or also don't want to give any permission to write, delete, create or do any thing my superset, i want to give the permission by which they can only see my interactive dashboard.

#apachesuperset
#apache superset
#permission

**Comment by LiteBrite82** [comment]
Hi @srishtiagrawall ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this software, such as the [apache/superset/discussions](https://github.com/apache/superset/discussions). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
